pssh.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, host,
         :param num_retries: (Optional) Number of retries for connection attempts\
         before the client gives up. Defaults to 3.
         :type num_retries: int
-        :param timeout: (Optional) Number of seconds to timout connection attempts\
+        :param timeout: (Optional) Number of seconds to timeout connection attempts\
         before the client gives up. Defaults to 10.
```

Comment:
I'm afraid not. 

Best there is is [autoattribute](http://sphinx-doc.org/ext/autodoc.html#directive-autoattribute) for class attributes but it doesn't work for function keyword arguments.